lazy_tensor_core/lazy_tensor_core/csrc/view_ops/opcode.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <torch/csrc/lazy/ts_backend/ts_node.h>
+
+namespace torch_lazy_tensors {
+namespace ir {
+namespace ops {
+
+// TODO: Allow other backend to define its own BaseNode, or
```

Comment:
so, the idea would be that we replace this part with a generated header, so that the IR codegen (which already knows about the custom backend node class) just stamps out the right `using` statement, right?

I was thinking it's nicer to leave this file as a non-generated one, and have it include the generated file- this is more discoverable to users...

But i'm afraid that's impossible unless we actually codegen this file too, since the include path would be different for xla backend and ts backend.  Unless there is some way to make the xla codegen (which runs out of tree) generate a header in the torch/csrc/lazy/core/ tree directly instead of in the torch_xla external tree?

I'm actually not sure how this would work for xla.  Guess we better figure it out before we go this route, generally